Svc/CmdSequencer/test/ut/CmdSequencerTester.hpp

```diff
@@ -174,45 +121,31 @@ namespace Svc {
             // ----------------------------------------------------------------------
 
             //! Enable the interceptor
-            void enable();
+            void enable(EnableType::t enableType);
 
             //! Disable the interceptor
             void disable();
 
-          private:
-
-            // ----------------------------------------------------------------------
-            // Private instance methods
-            // ----------------------------------------------------------------------
-
-            //! Intercept an open request
-            //! \return Success or failure
-            bool intercept(
-                Os::File::Status &fileStatus,
-                void *buffer,
-                NATIVE_INT_TYPE &size
-            );
+            class Override : public Os::Posix::File::PosixFile {
```

Comment:
Why is this class called Override? It seems better to give it a descriptive name based on what it does. You could call it PosixFile, if it's a specific implementation of Os::Posix::File::PosixFile.

Reply:
It is really an interceptor working with PosixFile so I will call it `PosixFileInterceptor`

Reply:
Sounds good!